app/marketing/management/commands/gdpr_reconsent_eu.py

```diff
@@ -0,0 +1,56 @@
+'''
```

Comment:
`'''` can have unexpected consequences in py files.  I've been trying to move to `"""` without the additional indent to pass pep-257 checks so we can ultimately auto-enforce this.